app/views/restaurant/show.html.haml

```diff
@@ -0,0 +1,34 @@
+.container
+  .row
+    .col
+    .col-8
+      %h1.extra-margin= @restaurant.name
+      %p.address
+        = @restaurant.street_address
```

Comment:
You can probbly use `@restaurant.full_address` and make this into one line. But even if you don't, you should be able to use the `.join()`method to fit this into one line instead of using interpolation across many lines.